classy/data/data_modules.py

```diff
@@ -196,14 +141,16 @@ def load_bundle(
                         train_coordinates.main_file_extension,
                     ) = load_bundle(
                         coordinates_dict.get("train_dataset"),
+                        task,
                         compute_main_extension=True,
                     )
                     logger.info(
                         f"Main extension found: {train_coordinates.main_file_extension}"
                     )
                 else:
                     train_coordinates.train_bundle = load_bundle(
-                        coordinates_dict.get("train_dataset")
+                        coordinates_dict.get("train_dataset"),
```

Comment:
shouldn't this become dataset in the new version?

Reply:
Nope this is only the training coordinates file

Reply:
mmm I'd like to be consistent, is it possible or does it break something?